webapp/public/app.css

```diff
@@ -0,0 +1,197 @@
+:root {
+  --polygon-path: 50% 0%, 100% 25%, 100% 75%, 50% 100%, 0% 75%, 0% 25%;
+  --cyan: #96D2D9;
+  --green: #14D990;
+  --red: #F24968;
+  --blue: #6929F2;
+  --yellow: #F2B807;
+  --magenta: #9B72F2;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: #112b3c;
+  --s: 100px;
+  --m: 4px;
+  font-family: monospace;
+  background-repeat: no-repeat;
+  background-image:
+    linear-gradient(45deg, var(--blue) 20%, var(--blue) 20%, rgba(255, 255, 255, 0) 20%, rgba(255, 255, 255, 0) 80%, var(--blue) 80%, rgba(255, 255, 255, 0) 100%),
+    linear-gradient(315deg, var(--yellow) 20%, var(--yellow) 20%, rgba(255, 255, 255, 0) 20%, rgba(255, 255, 255, 0) 80%, var(--yellow) 80%, rgba(255, 255, 255, 0) 100%),
+    linear-gradient(180deg, var(--green) 5%, var(--green) 5%, rgba(255, 255, 255, 0) 5%, rgba(255, 255, 255, 0) 95%, var(--green) 95%, var(--green) 100%);
+}
+
+.board {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 900px;
+  height: 900px;
+  font-size: 0;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  gap: 2px;
+  position: relative;
+  top: -20px;
+}
+
+.tile {
+  width: var(--s);
+  margin: var(--m);
+  height: calc(var(--s) * 1.1547);
```

Comment:
Magic number, please make it a const with an expressive name, and/or link to the computation for it